_includes/header.html

```diff
@@ -4,6 +4,17 @@
 
     {% assign custom_url = site.url | append: site.baseurl %}
     {% assign full_base_url = custom_url | default: site.github.url %}
+    {% if site.header_pages %}
+      {% assign header_pages = "" | split:"|"  %}
```

Comment:
I got this from https://talk.jekyllrb.com/t/how-do-you-add-items-to-an-array-in-jekyll/324/4 to initialize an empty array.